BEPs/BEP-341.md

```diff
@@ -0,0 +1,113 @@
+## 2. Abstract 
+BEP-341 describes a new allocation method for priority block-producing rights, which is logically concise and significantly enhances the system's transaction processing capacity, while remaining orthogonal to transaction processing optimization techniques within blocks.
+## 3. Motivation 
+The BSC ecosystem is active and continuously evolving, requiring ongoing improvements to the system's transaction processing capacity.
+## 4. Specification 
+### 4.1 Scaling Principle 
+Currently, each validator obtains priority block-producing rights for a single slot and is then rotated, with a fixed block interval of t. The transaction processing limit is t/2 for validating transactions from the previous block and t/2 for processing transactions in the new block.
+<div align="center">
+<img src=./assets/bep-341/4.1-1.png width=60% />
+</div>
+Assuming the number of transactions that can be processed is T, the average TPS is calculated as follows:
+<div align="center">
+<img src=./assets/bep-341/4.1-2.png width=18% />
+</div>
+After implementing this BEP, each validator can obtain priority block-producing rights for a continuous sequence of n slots in one round. The first block they process still allocates t/2 for validating transactions from the previous block and t/2 for processing transactions in the new block. However, subsequent blocks can skip the transaction validation process and focus on processing transactions in the new block. It is recommended that the processing time for the last block in their current round be only half of the time required to process transactions in the new block. This helps avoid the situation where the first block processed by the next validator is close to being an empty block.
+<div align="center">
+<img src=./assets/bep-341/4.1-3.png width=60% />
+</div>
+The TPS after continuous block production is as follows:
+<div align="center">
+<img src=./assets/bep-341/4.1-4.png width=50% />
+</div>
+Therefore, the TPS improvement rate is:
+<div align="center">
+<img src=./assets/bep-341/4.1-5.png width=40% />
+</div>
+The relationship between the number of continuous blocks produced and TPS improvement is 
+<div align="center">
+<img src=./assets/bep-341/4.1-6.png width=60% />
+</div>
+As shown in the graph, it is evident that continuous block production cannot double the TPS. There is no benefit when n<3, and increasing n beyond 5 does not proportionally increase the benefits. Therefore, a value within the range [3, 5] is suitable for n; when n=4, r=50%.
+
+### 4.2 Implementation Specification 
+#### 4.2.1 Priority Allocation 
+Each epoch predefines a set of validators, with a total of validatorN validators, and each validator within the set has a unique index ranging from [0, validatorN). If the current block height is blockN, then the validators with the following indices obtain priority block-producing rights.
```

Comment:
Could we elaborate a bit more here with an example? 
Say if blockN is 100, n = 4, validatorN is 10, which indices obtain priority? Also could you add brackets to make the formulas easier to understand?

Reply:
> Could we elaborate a bit more here with an example? Say if blockN is 100, n = 4, validatorN is 10, which indices obtain priority? Also could you add brackets to make the formulas easier to understand?

100/4%10 -->5
so the index is 5

Reply:
may not need these .png files, insert latex like: $\(blockN/n)\mod{validatorN}$